tests/window/resizing.rs

```diff
@@ -22,8 +22,11 @@ fn main() {
         .add_plugins(
             DefaultPlugins.set(WindowPlugin {
                 primary_window: Some(Window {
-                    resolution: WindowResolution::new(MAX_WIDTH as f32, MAX_HEIGHT as f32)
-                        .with_scale_factor_override(1.0),
+                    resolution: WindowResolution::new(LogicalSize::new(
```

Comment:
This also could be `WindowResolution::from((MAX_WIDTH as f32, MAX_HEIGHT as f32))`